src/Configurations.jl

```diff
@@ -150,6 +150,8 @@ Finds the distance between two positions a and b.
 """
 distance2(a,b) = (a-b)⋅(a-b)
 
+distance2(a,b,bc::SphericalBC) = (a-b)⋅(a-b)
+
 """
     distance2(a,b,bc::CubicBC) 
     
```

Comment:
please make the documentation string more general also including the other methods,